src/components/layout/LoadingSpinner/LoadingSpinner.jsx

```diff
@@ -0,0 +1,22 @@
+import styles from "./LoadingSpinner.module.css";
+
+const LoadingSpinner = () => {
+  return (
+    <div className={styles["container"]}>
+      <div className={styles["dot"]} style={{ "--i": 0 }}></div>
+      <div className={styles["dot"]} style={{ "--i": 1 }}></div>
+      <div className={styles["dot"]} style={{ "--i": 2 }}></div>
+      <div className={styles["dot"]} style={{ "--i": 3 }}></div>
+      <div className={styles["dot"]} style={{ "--i": 4 }}></div>
+      <div className={styles["dot"]} style={{ "--i": 5 }}></div>
+      <div className={styles["dot"]} style={{ "--i": 6 }}></div>
+      <div className={styles["dot"]} style={{ "--i": 7 }}></div>
+      <div className={styles["dot"]} style={{ "--i": 8 }}></div>
+      <div className={styles["dot"]} style={{ "--i": 9 }}></div>
+      <div className={styles["dot"]} style={{ "--i": 10 }}></div>
+      <div className={styles["dot"]} style={{ "--i": 11 }}></div>
```

Comment:
이거 갯수가 정해져있는 연산이라면 Array.from을 활용해서 코드를 간소화해보는건 어떨까요?

```suggestion
      {Array.from({ length: 12 }, (_, i) => (
        <div key={i} className={styles["dot"]} style={{ "--i": i }}></div>
      ))}
```